app/controllers/application_controller.rb

```diff
@@ -2165,10 +2165,10 @@ def visibility_box_edit
 
   def get_record_display_name(record)
```

Comment:
@h-kataria solution you gave was perfect one, however their were no change in behavior after changing https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/miq_ae_class_controller.rb#L2566

I have tried to change https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller/ci_processing.rb#L153 to suggested one and it was working fine for me, may be this is the breakthrough we were looking for, can you recheck this ! 

before change (L153),
```    record_name = get_record_display_name(element)```

After change (L153),
```    record_name = element.name```